testpilot/frontend/static-src/app/main.js

```diff
@@ -65,9 +65,11 @@ app.extend({
   },
 
   // Send webChannel message to addon, use a Promise to wait for the answer.
-  waitForMessage(type, data, timeout = 1000) {
+  waitForMessage(type, data, timeout = 5000) {
```

Comment:
I suspect, but can't be sure, that this was contributing to some of the "never finishes installing" errors on some machines. If the add-on took more than a second to finish installing & set up, the "Installing" barber poll button would never stop.